woocommerce/woocommerce-ios
# Order Editing: Update Country/State selected style

closes #5113 

# Why

Selecting a country or a state had the wrong style on the Edit Address screen. This PR makes sure to update that.

# How

Updated `TitleAndValue` row to receive a new `Value` type instead of a plain string. This Value type has to options, `placeholder` or `content`. Depending on the provided option the view will render the underlying text with a different style.

# Screenshots

Country Selected | State Selected
--- | ---
<img width="414" alt="first" src="https://user-images.githubusercontent.com/562080/135928420-7afe5397-7d30-4cf4-ba8e-8c2611121ea4.png"> | <img width="430" alt="second" src="https://user-images.githubusercontent.com/562080/135928422-847d9dc8-51b6-48b3-a747-0de7b522381a.png">

# Testing Steps

- Go to edit an order shipping address.
- See that when no country or state(where there is a list of states) is selected, there is a placeholder text with a subtle text style.
- See that when a country or a state is selected, the country/state name is rendered with a body text style.

Update release notes:
